docs/topics/compiler-reference.md

```diff
@@ -156,9 +156,15 @@ environment.
 
 Use a custom JDK home directory to include into the classpath if it differs from the default `JAVA_HOME`.
 
+### -Xjdk-release
```

Comment:
I guess it should be `-Xjdk-release=_version_`, similar to `-jvm-target _version_` below, since this option takes the same argument -- the version number.

(`=` is needed because `-X` options require it.)

Reply:
Whoops, just missed this, thanks!

Reply:
_ _ in _version_ is just a formating which is not working inheaders